src/components/Utilities/Unrecognized/AvDumpSeriesSelectModal.tsx

```diff
@@ -53,30 +64,56 @@ function AvDumpSeriesSelectModal({ getLinks, onClose, show }: Props) {
     handleSearch(commonSeries);
   }, [commonSeries, handleSearch]);
 
+  useLayoutEffect(() => () => {
```

Comment:
why are we using useLayoutEffect react docs recommend using useEffect

Reply:
Because the _effect_ needs to happen before the DOM paint, otherwise there will be visual glitches. I've tested it with both `useEffect` and `useLayoutEffect`, and when I used the former, then the visual glitches still remained.

Reply:
why is this `() => () => {`?
On line 59 too

Reply:
Because i want a cleanup effect. If you have a better way to write it that doesn't inteoduce the visual glitching again then please come with suggestions.

Reply:
Right... cleanup. I got confused because there was no actual effect, only cleanup. Have never written it that way